3.4.14/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre-slim
+FROM adoptopenjdk:latest
```

Comment:
should use tagged release to provide reproducible images

Reply:
good catch, thanks.  I pinned to 8u232-b09-jre-hotspot (commit: https://github.com/31z4/zookeeper-docker/pull/90/commits/c38c30182aa6b9a0566df809926141e217211e30) which also reduced image size by ~120MB per platform since their latest tag is the jdk instead of the jre.

Reply:
Also, we're sticking wit jre 8, right? I think it's better to switch to jre 11 later if there any benefits to do so.

Reply:
I don't see any arm images using jre 8 from adoptopenjdk:
https://hub.docker.com/_/adoptopenjdk?tab=tags&page=1&name=-b08-

They do have tar.gz files we could download and extract:
https://adoptopenjdk.net/releases.html

We could also keep things as they are for the zookeeper:latest tag but create another tag, maybe `jre11` and/or `jre13` and only those tags would be available for arm32/64/etc...?

Reply:
https://hub.docker.com/r/adoptopenjdk/openjdk8 different repo for jdk8

Reply:
Ah, thanks.  I don't see a stable tag from that repo that is a manifest covering amd64/arm64/....
https://hub.docker.com/r/adoptopenjdk/openjdk8/tags?page=1&name=debianslim-jre
The debianslim-jre tag isn't available for arm64, but their debianslim-jre-nightly is.